js/src/apbct-public.js

```diff
@@ -255,12 +275,18 @@ function apbct_collect_visible_fields( form ) {
 
 }
 
-function apbct_visible_fields_set_cookie( visible_fields_collection ) {
+function apbct_visible_fields_set_cookie( visible_fields_collection, form_id ) {
 
 	var collection = typeof visible_fields_collection === 'object' && visible_fields_collection !== null ?  visible_fields_collection : {};
 
-	ctSetCookie("apbct_visible_fields", JSON.stringify( collection ) );
-
+	if( + ctPublic.data__set_cookies === 1 ) {
+		for ( var i in collection ) {
+			var collectionIndex = form_id !== undefined ? form_id : i;
```

Comment:
`var collectionIndex = form_id !== undefined ? form_id : i;`

Equation in condition while declare a variable. I think that is too much for 1 line.